README.md

```diff
@@ -160,7 +160,7 @@ When using docker you don't need to have any of the Go development tools install
 For example, run the following command to build terraform in a linux-based container for macOS.
```

Comment:
Since we're updating the command, can we update the description as well? 
```suggestion
For example, run the following command to install the required tools and build terraform in a linux-based container for macOS.
```

Reply:
Fixed